bar_app/models.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+
+class Reference(models.Model):
+    """
+    Model representing a reference of beer.
+    """
+    id = models.AutoField(primary_key=True)
```

Comment:
You are overriding django's default behaviour by doing this. Django does that already, but with a BigAutoField instead!
Who knows if your bar meets high success ? ;)